src/test/Makefile-client.am

```diff
@@ -719,6 +726,15 @@ ceph_test_rgw_obj_LDADD = \
 ceph_test_rgw_obj_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_test_rgw_obj
 
+ceph_test_rgw_bencode_SOURCES = test/rgw/test_rgw_bencode.cc
+ceph_test_rgw_bencode_LDADD = \ 
+        $(LIBRADOS) $(LIBRGW) $(LIBRGW_DEPS) $(CEPH_GLOBAL) \
+        $(UNITTEST_LDADD) $(CRYPTO_LIBS) \
+        -lcurl -luuid -lexpat
+
+ceph_test_rgw_bencode_CXXFLAGS = $(UNITTEST_CXXFLAGS)
+bin_DEBUGPROGRAMS += ceph_test_rgw_bencode
```

Comment:
@cbodley Is the paramter "#define ENABLE_CLIENT 1" in src/acconfig.h must be added? I think this file acconfig.h is generated when run autogen.sh.